iothub/device/samples/PnpDeviceSamples/Thermostat/Models/TemperatureController.json

```diff
@@ -0,0 +1,60 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:com:example:TemperatureController;1",
+  "@type": "Interface",
+  "displayName": "Temperature Controller",
+  "description": "Device with two thermostats and remote reboot.",
+  "contents": [
+    {
+      "@type": [
+        "Telemetry",
+        "DataSize"
+      ],
+      "name": "workingSet",
+      "displayName": "Working Set",
+      "description": "Current working set of the device memory in KiB.",
+      "schema": "double",
+      "unit": "kibibyte"
```

Comment:
Wondering if this is a right value?

Reply:
yeah, it's 1024 bytes -> to differentiate that kilo is usually 1000 and not 1024: https://en.wikipedia.org/wiki/Kibibyte